test/langtools/tools/javac/versions/Versions.java

```diff
@@ -89,7 +89,8 @@ static enum SourceTarget {
         EIGHTEEN(false,  "62.0", "18", Versions::checksrc18),
         NINETEEN(false,  "63.0", "19", Versions::checksrc19),
         TWENTY(false,    "64.0", "20", Versions::checksrc20),
-        TWENTY_ONE(false,"65.0", "21", Versions::checksrc21);
+        TWENTY_ONE(false,"65.0", "21", Versions::checksrc21),
+        TWENTY_TWO(false,"66.0", "22", Versions::checksrc21);
```

Comment:
Should there be `checksrc22` instead of `checksrc21`? Or is that done later?

Reply:
Good catch. I have a refactoring of the test planned which will eliminate the explicit "checksrc$N" methods.